### PR TITLE
[TAS-1217] 🚸 Add default value to stakeholders

### PR DESCRIPTION
**Pull request by AuroraHuang22**
截圖 2024-02-15 下午5 11 51

### CODE REVIEW

File: components/IscnRegisterForm.vue [hunk 1]
```diff
@@ -1114,38 +1115,48 @@ export default class IscnRegisterForm extends Vue {
...
-      }],
-      url: [],
-      likerId: '',
-      authorDescription: 'ISCN owner',
+
+    if (this.address) {
+      try {
+        const { data } = await this.$axios.get(
+          getUserInfoMinByAddress(this.address),
+        )
```
Comment (williamchong) [inline components/IscnRegisterForm.vue:1137]: Will this throw 404 error, skipping the whole `iscnOwner` block if the address does not have likeId?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: components/IscnRegisterForm.vue (hunk 1 above)
Comment (AuroraHuang22) [reply components/IscnRegisterForm.vue]: [🎨 Use default info when likerid not found](https://github.com/likecoin/app-like-co/pull/440/commits/1f52588176c140fb6d65b6939d6182a5ea3228d0)

File: components/IscnRegisterForm.vue [hunk 2]
```diff
@@ -1114,38 +1115,52 @@ export default class IscnRegisterForm extends Vue {
...
+      // eslint-disable-next-line no-console
+      console.error(error)
+      return {
+        name: address,
+        wallet: [{ content: address, id: 1, type: 'like', isOpenOptions: false }],
+        url: [],
+        likerId: '',
+        authorDescription: 'Publisher',
+      }
     }
```
Comment (williamchong) [inline components/IscnRegisterForm.vue:1162]:
```suggestion
    let userData: any = null;
    try {
      ({ data: userData } = await this.$axios.get(getUserInfoMinByAddress(address)))
    } catch (error) {
      // eslint-disable-next-line no-console
      console.error(error)
    }
    return {
      name: userData?.displayName || address,
      wallet: [{ content: address, id: 1, type: 'like', isOpenOptions: false }],
      url: [],
      likerId: userData?.user || '',
      authorDescription: userData?.description || 'Publisher',
    }
```

**Review by williamchong — CHANGES_REQUESTED** [review]
Also we can consider using `catchAxiosError`, but I think even non-404 error should be caught in this case

File: components/IscnRegisterForm.vue (hunk 2 above)
Comment (AuroraHuang22) [reply components/IscnRegisterForm.vue]: [🎨 Handle default info when user data is unavailable](https://github.com/likecoin/app-like-co/pull/440/commits/4474a883bc8b1d79c1d50380cd17c25a05fdfcfa)

**Pull request merged**